README.md

```diff
@@ -268,6 +271,7 @@ The elements can be specified using [FHIRPath](http://hl7.org/fhirpath/) syntax.
 |keep|All elements| Retains the value as is. |
 |redact|All elements| Removes the element. See the parameters section below to handle special cases.|
 |dateShift|Elements of type date, dateTime, and instant | Shifts the value using the [Date-shift algorithm](#date-shift-algorithm).
+|cryptoHash|All elements| Transforms the value using [Crypto-hash method](#Crypto-hash-method). |

```

Comment:
Make sure the hyperlink works. I could not verify it.

Reply:
Sure. I have verified that it's working https://github.com/microsoft/FHIR-Tools-for-Anonymization/tree/personal/yufei/cryptohash-readme#Crypto-hash-method